test/functorch/test_ops.py

```diff
@@ -1517,6 +1519,9 @@ def reference(primals, cotangents, primals_tangents, cotangents_tangents):
         xfail("_native_batch_norm_legit"),
         xfail('native_dropout_backward'),
         xfail('nn.functional.prelu'),
+
+        xfail('CubeGenVmapAutogradFunction'),  # NYI
+        xfail('SortGenVmapAutogradFunction'),  # https://github.com/pytorch/pytorch/issues/90067
```

Comment:
This has been fixed now! https://github.com/pytorch/pytorch/issues/90067

Reply:
Thank you, that was quick

Reply:
Updated to NYI (this PR doesn't add jvp support for generate_vmap_rule=True)

Reply:
By the way, I'm not immediately to remove the xfail in https://github.com/pytorch/pytorch/blob/f02e93b584ff77519af10f66ad5cfa644da19428/test/functorch/test_ops.py#L1360 after your PR.

The reason being is that it looks like it creates an torch.int64 tangent tensor instead of passing None: https://github.com/pytorch/pytorch/blob/f02e93b584ff77519af10f66ad5cfa644da19428/torch/testing/_internal/autograd_function_db.py#L238-L239 .

ctx.set_materialize_grads does make the problem go away, but... (1) morally I feel like there should never be a non-fp tangent tensor and (2) the set_materialize_grads documentation doesn't say anything about this case

Reply:
Yeah that looks l like a bug, taking a look.

Reply:
Should be fixed by https://github.com/pytorch/pytorch/pull/91183